airgun/entities/repository.py

```diff
@@ -14,20 +14,24 @@
 
 
 class RepositoryEntity(BaseEntity):
-    def _find_default_http_proxy(self, values):
-        """ Finds the global proxy name and returns updated values accordingly."""
-        proxy = SettingsEntity(self.browser)
-        result = proxy.read(property_name='name = content_default_http_proxy')['table'][0]['Value']
-        global_proxy_name = result.split()[0]
-        if global_proxy_name == 'no':
-            global_proxy_name = 'None'
-        values["repo_content.http_proxy_policy"] = f'Global Default ({global_proxy_name})'
-        return values
+    def _global_default_http_proxy(self):
+        """Look up the default http proxy and return the string that a user would select for
+        HTTP Proxy Policy when creating or updating a repository.
+        """
+        proxy_name = SettingsEntity(self.browser).read(
+            property_name='name = content_default_http_proxy'
+        )['table'][0]['Value']
+
+        # The default text for no default http proxy varies between versions of Satellite
+        if proxy_name in ('Empty', 'no global default'):
+            proxy_name = 'None'
```

Comment:
Not really related to the change you're making as this is the same in the old function... but a _string_ `'None'`?? *WHY???*

Reply:
That's how it shows up in the dropdown selector if no global default http proxy has been configured: 'Global Default (None)'.